src/test/correct/arrays_simple/clang/arrays_simple.expected

```diff
@@ -1,37 +1,60 @@
 var {:extern} Gamma_R0: bool;
+
 var {:extern} Gamma_R31: bool;
+
 var {:extern} Gamma_R8: bool;
+
 var {:extern} Gamma_mem: [bv64]bool;
+
 var {:extern} Gamma_stack: [bv64]bool;
+
 var {:extern} R0: bv64;
+
 var {:extern} R31: bv64;
+
 var {:extern} R8: bv64;
+
 var {:extern} mem: [bv64]bv8;
+
 var {:extern} stack: [bv64]bv8;
+
 const {:extern} $_IO_stdin_used_addr: bv64;
 axiom ($_IO_stdin_used_addr == 1872bv64);
+
 function {:extern} {:bvbuiltin "bvadd"} bvadd64(bv64, bv64) returns (bv64);
+
 function {:extern} gamma_load32(gammaMap: [bv64]bool, index: bv64) returns (bool) {
-  (gammaMap[bvadd64(index, 3bv64)] && (gammaMap[bvadd64(index, 2bv64)] && (gammaMap[bvadd64(index, 1bv64)] && gammaMap[index])))
+  (gammaMap[bvadd64(index, 3bv64)]&&(gammaMap[bvadd64(index, 2bv64)]&&(gammaMap[bvadd64(index, 1bv64)]&&gammaMap[index])))
```

Comment:
What's caused this to change? I haven't played around with it yet but I can't obviously see a cause yet because it doesn't seem to happen when && is used for Gamma assignments. It does also happens with ++ in memory_loads though.

Reply:
Oh the && in Gamma assignments is unrelated, so my push fixes this